### PR TITLE
Adding label research page

### PR DESCRIPTION
**Pull request by sopranopillow**
Add research page for the `Label` component.

### CODE REVIEW

File: research/src/pages/label.research.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+menu: Analysis
```
Comment (gregwhitworth) [inline research/src/pages/label.research.mdx:2]: A PR that landed just before yours modified this to 'Research' so please update

**Review by gregwhitworth — CHANGES_REQUESTED** [review]

**Comment by andrico1234** [comment]
Hi @sopranopillow , just bumping this PR. are you happy for a re-review?

**Comment by sopranopillow** [comment]
> Hi @sopranopillow , just bumping this PR. are you happy for a re-review?

For sure!

**Comment by gregwhitworth** [comment]
Closing this due to the lack of activity, it can always be re-opened in the future.

**Pull request closed without merging**